pygmt/src/grdgradient.py

```diff
@@ -0,0 +1,116 @@
+"""
+grdgradient - Create a gradient from a grid.
+"""
+
+import xarray as xr
+from pygmt.clib import Session
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import (
+    GMTTempFile,
+    args_in_kwargs,
+    build_arg_string,
+    fmt_docstring,
+    kwargs_to_strings,
+    use_alias,
+)
+
+
+@fmt_docstring
+@use_alias(
+    A="azimuth",
+    D="direction",
+    E="radiance",
+    G="outgrid",
+    R="region",
+    V="verbose",
+)
+@kwargs_to_strings(
+    A="sequence",
+    R="sequence",
+)
+def grdgradient(grid, **kwargs):
+    r"""
+    Compute the directional derivative in a given ``azimuth`` or
+    ``direction`` of the vector gradient of the data.
```

Comment:
```suggestion
    Compute the directional derivative in a given ``azimuth`` or
    ``direction`` of the vector gradient of the data.
```
Could you shorten this description or split it? Currently this description seems to be too long for a compact view in the overview table under *API Reference*.

Reply:
Split it up in 0d26dc1; hope it looks good!